src/components/Message/Message.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Img, Text } from '../styles/UI';
+import { useRecoilState } from 'recoil';
+import { UserState } from '../../atoms';
+
+export default function Message({ message, isOwner }) {
+  const [users, setUser] = useRecoilState(UserState);
+
+  const user = users.find((u) => u.userId === message.userId);
```

Comment:
`user`와 같은 명시적인 이름을 써주면 좋을 것 같아요!
```suggestion
  const user = users.find((user) => user.userId === message.userId);
```